src/flows/pipes/Schedule/view.tsx

```diff
@@ -143,11 +143,11 @@ const Schedule: FC<PipeProp> = ({Context}) => {
   let intervalError = ''
   let offsetError = ''
 
-  if (!data.interval) {
+  if (data?.interval === '') {
```

Comment:
are these different?

Reply:
They are. The previous implementation had an undefined value so it was automatically registering an error when the user created a task but didn't yet input a change